src/Microsoft.ML.StandardLearners/Standard/MultiClass/MetaMulticlassTrainer.cs

```diff
@@ -24,7 +24,7 @@ public abstract class ArgumentsBase
         {
             [Argument(ArgumentType.Multiple, HelpText = "Base predictor", ShortName = "p", SortOrder = 1, SignatureType = typeof(SignatureBinaryClassifierTrainer))]
             [TGUI(Label = "Predictor Type", Description = "Type of underlying binary predictor")]
-            public IComponentFactory<TScalarTrainer> PredictorType;
```

Comment:
See the comment here [https://github.com/dotnet/machinelearning/pull/622#discussion_r207046122](https://github.com/dotnet/machinelearning/pull/622#discussion_r207046122) why I didn't do it this way to begin with.  I don't think we should be closing off this interface to a sub-interface, because it limits what kinds of classes can implement the interface.